code/modules/ninja/suit/n_suit_verbs/energy_net_nets.dm

```diff
@@ -9,20 +9,26 @@ It is possible to destroy the net by the occupant or someone else.
 	icon = 'icons/effects/effects.dmi'
 	icon_state = "energynet"
 
-	density = TRUE//Can't pass through.
-	opacity = 0//Can see through.
-	mouse_opacity = MOUSE_OPACITY_ICON//So you can hit it with stuff.
-	anchored = TRUE//Can't drag/grab the net.
+	density = TRUE //Can't pass through.
+	opacity = 0 //Can see through.
```

Comment:
false

Reply:
http://www.byond.com/docs/ref/info.html#/atom/var/opacity
Name is a bit misleading